### PR TITLE
Added missing space to match PeopleSidebar

### PR DESCRIPTION
**Pull request by rawnsley**
Apologies for being pedantic, but it's been bothering me for a while.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-791)

### CODE REVIEW

**Comment by yakyouk** [comment]
> 
> 
> Apologies for being pedantic, but it's been bothering me for a while.

haha, was bothering me too.
Suggesting this:
```
        <FormattedMessage
          id="content-menu.people-menu-button"
          defaultMessage="People ({presenceCount})"
          values={{ presenceCount: props.presenceCount }}
        />
```

**Comment by netpro2k** [comment]
I agree with @yakyouk's suggestion. Either that or adding the space outside of the translated message. Otherwise all translation will need to remember to include this trailing space.

**Comment by rawnsley** [comment]
> I agree with @yakyouk's suggestion. Either that or adding the space outside of the translated message. Otherwise all translation will need to remember to include this trailing space.

Agreed and updated. Thanks @yakyouk

File: src/react-components/room/ContentMenu.js [hunk 1]
```diff
@@ -36,8 +36,7 @@ export function PeopleMenuButton(props) {
     <ContentMenuButton {...props}>
       <PeopleIcon />
       <span>
-        <FormattedMessage id="content-menu.people-menu-button" defaultMessage="People ({presenceCount})" values={{ presenceCount: props.presencecount }}
-        />
+        <FormattedMessage id="content-menu.people-menu-button" defaultMessage="People ({presenceCount})" values={{ presenceCount: props.presencecount }} />
```
Comment (yakyouk) [inline src/react-components/room/ContentMenu.js:39]: It seems that when the file gets formatted with prettier, it gets broken again into multiple lines due to [prettier repo settings](https://github.com/mozilla/hubs/blob/a8eb03573f4e195f7a587a2964be03081b512092/.prettierrc.json#L2)

Comment (rawnsley) [reply src/react-components/room/ContentMenu.js:39]: Huh. When does that get enforced? I'm not seeing it in VSCode, but maybe I've turned that option off. Either way I can change it to fit. This appears to be the canonical style for split-line elements:
```
      <span>
        <FormattedMessage 
          id="content-menu.people-menu-button" 
          defaultMessage="People ({presenceCount})" 
          values={{ presenceCount: props.presencecount }} 
        />
      </span>
```

Comment (rawnsley) [reply src/react-components/room/ContentMenu.js]: I've installed Prettier and applied it to the file. Thanks for the info.

**Review by johnshaughnessy — APPROVED** [review]

**Comment by Softvision-GeluHaiduc** [comment]
I have verified this using Windows 10 on the latest releases for Firefox and Chrome browsers, I tested using different languages. 
While testing I encountered 2 new issues that I logged here #4646 and #4647

**Pull request merged**